docs/sources/administration/roles-and-permissions/access-control/rbac-fixed-basic-role-definitions/index.md

```diff
@@ -159,9 +160,10 @@ This will allow you to control access to different OnCall features using the fol
 
 The following table lists the default RBAC OnCall role assignments to the basic roles:
 
-| Basic role    | Associated fixed roles              | Description                                                                                                        |
-| ------------- | ----------------------------------- | ------------------------------------------------------------------------------------------------------------------ |
-| Grafana Admin | `plugins:grafana-oncall-app:admin`  | Default [Grafana server administrator]({{< relref "../#grafana-server-administrators" >}}) assignments.            |
-| Admin         | `plugins:grafana-oncall-app:admin`  | Default [Grafana organization administrator]({{< relref "../#organization-users-and-permissions" >}}) assignments. |
-| Editor        | `plugins:grafana-oncall-app:editor` | Default [Editor]({{< relref "../#organization-users-and-permissions" >}}) assignments.                             |
-| Viewer        | `plugins:grafana-oncall-app:reader` | Default [Viewer]({{< relref "../#organization-users-and-permissions" >}}) assignments.                             |
+| Basic role    | Associated fixed roles              | Description                                                                                             |
+| ------------- | ----------------------------------- | ------------------------------------------------------------------------------------------------------- |
+| Grafana Admin | `plugins:grafana-oncall-app:admin`  | Default [Grafana server administrator]({{< relref "../#grafana-server-administrators" >}}) assignments. |
+| Admin         | `plugins:grafana-oncall-app:admin`  | Default [Grafana organization administrator]({{< relref "../#basic-roles" >}}) assignments.             |
+| Editor        | `plugins:grafana-oncall-app:editor` | Default [Editor]({{< relref "../#basic-roles" >}}) assignments.                                         |
+| Viewer        | `plugins:grafana-oncall-app:reader` | Default [Viewer]({{< relref "../#basic-roles" >}}) assignments.                                         |
+| None          | `plugins:grafana-oncall-app:none`   | Default [None]({{< relref "../#basic-roles" >}}) assignments.                                           |
```

Comment:
I don't think OnCall define the role none yet.
```suggestion
```

Reply:
Should OnCall have the role defined?